crates/ruma-events/src/call/member.rs

```diff
@@ -0,0 +1,386 @@
+//! Types for matrix rtc state events ([MSC3401]).
+//!
+//! This implements a newer/updated version of MSC3401
+//! [MSC3401](https://github.com/matrix-org/matrix-spec-proposals/pull/3401)
```

Comment:
```suggestion
//! This implements a newer/updated version of MSC3401.
//!
//! [MSC3401]: https://github.com/matrix-org/matrix-spec-proposals/pull/3401
```

The docs CI fails because of this.

Reply:
do you know what exactly is wrong with it?

Reply:
Actually the `MSC3401` should be in square brackets on both lines.

About what's wrong with it: You wrote a literal `MSC3401` followed by a linkified `MSC3401`, as in:

MSC3401 [MSC3401](https://github.com/matrix-org/matrix-spec-proposals/pull/3401)

Reply:
Actually what the CI complained about is that there is a link defined on line 1, because of `[MSC3401]`, that has no URL

Reply:
Initially i copied over this:
```
//! Types for extensible audio message events ([MSC3927]).
//!
//! [MSC3927]: https://github.com/matrix-org/matrix-spec-proposals/pull/3927
```
like so:
```
//! Types for extensible audio message events ([MSC3401]).
//!
//! [MSC3401]: https://github.com/matrix-org/matrix-spec-proposals/pull/3401
```
But this did not make the ci happy that is why I started to do crazy things to see if i can at least make it pass

Reply:
I think you made a small mistake when copy-pasting but fixed it the wrong way and made things worse 😅